dask_jobqueue/__init__.py

```diff
@@ -1,13 +1,14 @@
 # flake8: noqa
 from . import config
-from .core import JobQueueCluster
-from .moab import MoabCluster
-from .pbs import PBSCluster
-from .slurm import SLURMCluster
-from .sge import SGECluster
-from .lsf import LSFCluster
-from .oar import OARCluster
-from .htcondor import HTCondorCluster
+from .local import LocalJob, LocalCluster
```

Comment:
I would rather not have `LocalCluster` and `LocalJob` available at the root package level. That could create some confusion with `distributed.LocalCluster`. Do you see a use case for dask_jobqueue.local.LocalCluster outside of tests?

I am slightly inclined to also leave all the job classes outside of `dask_jobqueue` namespace as well since they are implementation details. Also slightly annoying to have tab-completion not work as well:
```py
# this gives two solutions: SGEJob and SGECluster instead of immediately completing
from dask_jobqueue import SGE<TAB>
```

If you agree this is something I can definitely do myself when I am feeling not too productive.